### PR TITLE
first go at reworking private data section into discrete principles

### PR DESCRIPTION
**Pull request by pes10k**
For discussion on the Privacy Task Force call today.  The text would need a lot of polish, but eager to get feed back on the effort to reorg, etc

***
<a href="https://pr-preview.s3.amazonaws.com/w3ctag/privacy-principles/pull/133.html#data" title="Last updated on Apr 20, 2022, 11:04 PM UTC (5b39144)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3ctag/privacy-principles/133/ff66160...5b39144.html" title="Last updated on Apr 20, 2022, 11:04 PM UTC (5b39144)">Diff</a>

### CODE REVIEW

**Comment by pes10k** [comment]
As mentioned briefly at the end of the call, this isn't something anyone would want to merge as is (and a lot of the text that currently commented out would be good to work back in), but its an effort to i) merge the two sections and ii) focus the section around specific principals

File: index.html [hunk 1]
```diff
@@ -849,10 +849,65 @@
...
+not appropriate for a data-collecting site to infer those goals in broad
+and abstract ways (e.g., its not appropriate for a site author to reason
+"if the user visits my site, then it must be their goal to support the site;
+therefor the user wants me to collect their personal data for monetization
+purposes.").
+
+
+### Data Can "Personal Data" Even If It Doesn't Enable Identification
+
+Unwanted collection of "personal data" is harmful to people, even when
```
Comment (samuelweiler) [inline index.html:881]: This is related to data about others (PR#130).  how should this be tied to PR130?

File: index.html [hunk 2]
```diff
@@ -849,10 +849,65 @@
   This section has not been edited for narrative.
 </aside>
 
-We define <dfn data-lt="data">personal data</dfn> as any information that is directly or
-indirectly related to an identified or identifiable [=person=], such as by reference to an
-[=identifier=] ([[GDPR]], [[OECD-Guidelines]], [[Convention-108]]).
+Personal data is data describing a person, relating to that persons interests,
```
Comment (samuelweiler) [inline index.html:852]: s/persons/person's/

File: index.html [hunk 3]
```diff
@@ -1023,150 +1026,66 @@
...
 
-* The <dfn data-export="">right to access</dfn> [=data=] about oneself.
+<div class="practice">
 
-This right includes being able to discover data about oneself (implying no databases are kept
-secret) and to review
-what information has been collected or inferred.
+<p><span class="practicelab" id="user-agent-data-sharing">
+User agents should only share and present information about the user that
+the user can reasonably anticipate being shared, and which directly relate to the users overt, immediate goals.
```
Comment (jyasskin) [inline index.html:1053]: Some folks pointed out that "directly relate to ... overt, immediate goals" is probably too limiting. Users have privacy goals that are longer-term than immediate, for example avoiding the need to log in with PII, and we shouldn't have this document claim that short-term privacy goals always trump long-term ones.

File: index.html [hunk 4]
```diff
@@ -1023,150 +1026,66 @@
...
-data on their device, on a server, or both, and the distinctions may not always be clear.
+Examples:
 
-* The <dfn data-export="" data-lt="right to portability" data-local-lt="portability">right to
-port</dfn> [=data=], including data one has stored with a [=party=], so it can easily be reused or
-transferred elsewhere.
+* Sharing geo-location data with a map website after a user has clicked a "share my location" button has
+  a small number of "hops" between intent and data.
+* Sharing information about the user's selected DNS resolver with a site, so that the site authors can debug
+  possible issues around tail/uncommon resolvers, has a large number of "hops" between user intent (to do
```
Comment (jyasskin) [inline index.html:1066]: Maybe

```suggestion
  possible issues around tail/uncommon resolvers, has more "hops" between user intent (to do
```

Users intend to, well, _use_ the websites they're visiting, and buggy websites prevent that.

File: index.html (hunk 3 above)
Comment (pes10k) [reply index.html:1053]: I do not think i follow.  The "users overt, immediate goals" text is about what the user intends to do on the page or how they intend to interact w/ the site, not the user's privacy goals. Am i misunderstanding the concern?

File: index.html (hunk 4 above)
Comment (pes10k) [reply index.html:1066]: Can you say more about how `a large number of` -> `more` connects to below concern?

> Users intend to, well, use the websites they're visiting, and buggy websites prevent that.

W/o agreeing or disagreeing, im not sure I understand the connection between the two.  Can you say more?

**Pull request merged**